client/mysqltest.cc

```diff
@@ -4159,7 +4159,13 @@ void do_perl(struct st_command *command)
       die("Failed to create temporary file for perl command");
     my_close(fd, MYF(0));
 
-    str_to_file(temp_file_path, ds_script.str, ds_script.length);
+    /* Compatibility for Perl 5.24 and newer. */
+    static DYNAMIC_STRING script;
+    init_dynamic_string(&script, "push @INC, \".\";\n", 1024, 1024);
```

Comment:
Would it not be better to move push @INC, "." to db_stript initialization, i.e. init_dynamic_string(&ds_script, "", 1024, 1024); and abandon script variable ?